src/query/functions/tests/it/scalars/testdata/string.txt

```diff
@@ -2430,7 +2430,7 @@ evaluation:
 +--------+-----------------+------------------+--------------+
 | Type   | String          | String           | Int8         |
 | Domain | {"cc"..="test"} | {"ccb"..="test"} | {-128..=127} |
-| Row 0  | 'hi'            | 'i'              | 1            |
+| Row 0  | 'hi'            | 'i'              | -1           |
```

Comment:
@sundy-li I think the new one is correct. Please confirm.

Reply:
right!